### PR TITLE
uk-UA: Added translation

### PR DESCRIPTION
**Pull request by AyzekUorren**
Ukrainian translation

### CODE REVIEW

**Comment by AyzekUorren** [comment]
<details>
<summary>Screenshots</summary>

image
image
image
image
image
image
image
image
image
image

</details>

**Comment by Amrsatrio** [comment]
Can you shorten "Restart File Explorer"? In Japanese I removed "File" to shorten it.

**Comment by AyzekUorren** [comment]
Yes, sure.

**Comment by Amrsatrio** [comment]
Please translate the remaining English texts first, then I will merge once they have been translated.

File: resources/lang/ep_gui.uk-UA.rc [hunk 1]
```diff
@@ -29,8 +29,8 @@ BEGIN
...
     IDS_CRASH_ONCE              "Один раз"
```
Comment (Amrsatrio) [inline resources/lang/ep_gui.uk-UA.rc:34]: Change to lowercase first letter because it gets substituted into %1$s

Comment (AyzekUorren) [reply resources/lang/ep_gui.uk-UA.rc]: I've added changes, please review if possible.
Additionally, I fixed several errors that I found.

**Pull request closed without merging**